packages/langium/test/utils/uri-utils.test.ts

```diff
@@ -21,18 +22,36 @@ describe('URI Utils', () => {
         expect(UriUtils.relative(from, to)).toBe('../d.txt');
     });
 
+    test.skipIf(process.platform !== 'win32')('relative path in parent directory win32', () => {
```

Comment:
FYI, feel free to run these tests on Windows as well - `URI` is capable of reading windows style file paths even on non-windows systems :)

Reply:
Huh, that's unexpected. I don't like that, but it seems like you're right :)